elections/election-2022.md

```diff
@@ -0,0 +1,102 @@
+# Project nomination for BSC representatves 2022
+
+In accordance with the [Elections Schedule](./schedule.md) and the
+[eBPF Foundation Charter][charter], this page holds a public record of the
+steps undertaken to nominate members for the BSC in 2022.
+
+The following members' terms will expire in 2022. The Section 4.b column
+indicates which projects must be considered to draw nominations from.
+
+| Name    | Section 4.b  |
+| ------- | ------------ |
+| (empty) | eBPF runtime |
+| TBD     |              |
+
+## Nominations (ETA August)
+
+The following nominees are under consideration for election to the BSC.
+
+### Kernel Candidates
+
+_[eBPF Foundation Charter][charter] §4.b.i Two Linux Kernel representatives
+from the group of eBPF Linux kernel maintainers (as specified in the Linux eBPF
+maintainer file). The Linux Kernel representatives must not be employed by the
+same Company or by Related Companies._
+
+* TBD
+
+### Runtime Candidates
+
+_[eBPF Foundation Charter][charter] §4.b.ii Up to two representatives from
+other open source eBPF runtime implementations.￼Each “eBPF Runtime
+Representative” will represent a different eBPF runtime chosen by the BSC, and
+must be an active contributor to that eBPF runtime._
+
+* eBPF Runtime for Windows
+
+### Technical Project Candidates
+
+_[eBPF Foundation Charter][charter] §4.b.iii Up to three technical project
+representatives. Each Technical Project Representative candidate must be an
+active contributor to a different Technical Project chosen by the BSC._
+
+* TBD
```

Comment:
I am hoping several of the projects in lines 52-78 will move up here in the near future.  (E.g., the following that are part of IoVisor today: BCC, bpftrace, kubectl trace, ply)

Reply:
+1 I realized also that given we've changed the charter, there is currently a bit of an odd mismatch between the prior appointments and the current categories. For instance, since this section was previously about "maintainers" of community projects, Andrii, Brendan and I are technically part of the category that this section became. Brendan may still fall under this category given his relationship with some of those projects you mentioned, but we have not discussed this at all for Katran/Cilium . As a Cilium representative, I wouldn't be averse to submitting Cilium under this process as well, assuming that its relationship with CNCF doesn't make that more muddy. But I'll probably have to follow up with some others about whether that is a reasonable and viable thing to do.

Reply:
I've annotated this in the latest version.